assets/admin/students/student-action-menu/index.js

```diff
@@ -68,12 +70,20 @@ export const StudentActionMenu = ( {
 	 *
 	 * @return {Array} Filtered controls.
 	 */
-	const controls = applyFilters(
-		'senseiStudentActionMenuControls',
-		defaultControls,
-		setAction,
-		setModalOpen
-	);
+	useEffect( () => {
+		async function getMenuControls() {
+			const response = await applyFiltersAsync(
```

Comment:
Although this code works, I have no idea why or how. 😅  It looks good in theory, but when setting breakpoints or adding log statements in the code, I noticed that when the `fetch` in Sensei Pro returns and `setControls` is called below, the component does not re-render. Despite this, the "Add to Group" and "Remove from Group" options are added in the UI. 🤯 

Given I don't understand what is happening here, I'm somewhat averse to shipping this change.

Reply:
Maybe I'm missing something, but I didn't find where the `applyFiltersAsync` comes from. Checking the [Gutenberg package](https://github.com/Automattic/gutenberg/blob/trunk/packages/hooks/src/index.js), it doesn't seem to be there.

I'm also seeing this error in my browser console: `Uncaught (in promise) TypeError: (0 , _wordpress_hooks__WEBPACK_IMPORTED_MODULE_2__.applyFiltersAsync) is not a function`.

So I don't understand how the buttons are displayed for you. For me it's not being displayed. Maybe there is some other code adding them for you given a specific condition?

Reply:
It's in `@wordpress/hooks`, so you'll need do reinstall `npm` packages as I updated the version.

Reply:
> Are you using a WP beta version?

Oof, yes I am. I wasn't able to downgrade in the Beta Tester plugin so had to stay on 6.7. Back to square one then I guess. 😄  Thanks for taking a look!